life_expectancy/.gitignore

```diff
@@ -160,4 +160,8 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
\ No newline at end of file
+#.idea/
+
+
+# DS_Store
+.DS_Store
```

Comment:
The reason why your `.gitignore` file didn't work, is probably because git was expecting to see it in the root directory and it currently is inside a folder.

By the way, `.gitignore` will prevent new files and changes from being added, but it won't remove any file that had been previously added. To do so, you must manually run `git rm <FILE-PATH>`

Reply:
Ok, i need to think about this with more time latter. Tks!

Reply:
actually i have this file in both folders "life_expectancy" and "assignments". I wil delete the one inside life_expectancy and leave the one in the assignments root directory.